test/integration/default/default_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'pathname'
 
+ENV['PATH'] = 'C:\\msys2\\bin;C:\\msys2\\mingw32\\bin;C:\\msys2\\usr\\bin' if os[:family] == 'windows'
```

Comment:
https://github.com/chef-cookbooks/omnibus/pull/170 is using `['C:\msys2\bin', 'C:\msys2\mingw32\bin']` for 32bit and `['C:\msys2\bin', 'C:\msys2\mingw64\bin']` for 64bit. Does that mean that we need to add the 64 bin or remove the 32 bin here?